### PR TITLE
Add docs and tests

### PR DESCRIPTION
**Pull request by fhunleth**
This adds docs for the IPv4 config helper. The CommandRunner module is also tested now. It was partially tested indirectly via InterfaceTest.

### CODE REVIEW

**Comment by fhunleth** [comment]
@ConnorRigby When I was documenting the IPv4 config helper, I noticed that setting the default gateway won't work. The default gateway needs to be set on the RouteManager so that it can handle prioritizing it against other interfaces. I'm going to add this as an issue, but wanted to make you aware in case you were wondering why the Internet wasn't working...

File: lib/vintage_net/ip/config_to_interfaces.ex [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
+  * `:method` - `:dhcp` or `:static`
+
+  If `method: :static`, then the following addition fields are checked:
+
+  * `:address` - IPv4 address as a string
+  * `:netmask` - IPv4 netmask as a string
+  * `:broadcast` - IPv4 broadcast address as a string
+  * `:metric` - Route metric (TODO: THIS WON'T WORK)
+  * `:gateway` - Default gateway (TODO: THIS WON'T WORK)
+  * `:pointopoint` - ?
```
Comment (mattludwigs) [inline lib/vintage_net/ip/config_to_interfaces.ex]: Are you looking for information here about what this is? If so, is this helpful?

http://manpages.ubuntu.com/manpages/trusty/man5/interfaces.5.html

```
pointopoint address
  Address of other end point (dotted quad). Note the spelling of "point-to".
```

Comment (fhunleth) [reply lib/vintage_net/ip/config_to_interfaces.ex]: Yeah, I was lazy. I wonder if that works...

Comment (fhunleth) [reply lib/vintage_net/ip/config_to_interfaces.ex]: Fixed

**Pull request merged**